editor/src/uuiui/styles/theme/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Theme } from '../../../components/editor/store/editor-state'
+import { useEditorState } from '../../../components/editor/store/store-hook'
+import { dark } from './dark'
+import { light } from './light'
+import { ColorTheme, colorTheme, darkColorTheme, UtopiaTheme } from './utopia-theme'
+import type { UtopiColor } from '../utopi-color-helpers'
+
+export const colorThemeCssVariables = Object.keys(light).reduce((prev, curr) => {
+  const parts = curr.replace(/([A-Z]|[0-9]+)/g, '-$1'),
+    varName = `--${parts.toLowerCase()}`
```

Comment:
```suggestion
  const parts = curr.replace(/([A-Z]|[0-9]+)/g, '-$1')
  const varName = `--${parts.toLowerCase()}`
```
Please don't use the combined initialisation syntax for variables